hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/BlockCacheFactory.java

```diff
@@ -200,15 +209,39 @@ private static BlockCache createExternalBlockcache(Configuration c) {
 
   }
 
-  private static BucketCache createBucketCache(Configuration c) {
-    // Check for L2.  ioengine name must be non-null.
-    String bucketCacheIOEngineName = c.get(BUCKET_CACHE_IOENGINE_KEY, null);
+  private static BucketCache createBucketCache(Configuration c, CacheLevel level) {
+    // Check for ioengine name must be non-null.
+    String bucketCacheIOEngineName;
+    int writerThreads;
+    int writerQueueLen;
+    String persistentPath;
+    switch(level) {
+      case L1:
+        bucketCacheIOEngineName = c.get(CompositeBucketCache.IOENGINE_L1, null);
+        writerThreads = c.getInt(CompositeBucketCache.WRITER_THREADS_L1,
+            DEFAULT_BUCKET_CACHE_WRITER_THREADS);
+        writerQueueLen = c.getInt(CompositeBucketCache.WRITER_QUEUE_LENGTH_L1,
+            DEFAULT_BUCKET_CACHE_WRITER_QUEUE);
+        persistentPath = c.get(CompositeBucketCache.PERSISTENT_PATH_L1);
+        break;
+      case L2:
+      default:
```

Comment:
We need to LOG that we are creating both L1 and L2 caches as bucket cache.

Reply:
Add some log like this:
LOG.info("Creating BucketCache for {}, ioengine : {}, cacheSize {}.", level, bucketCacheIOEngineName, bucketCacheSize);